src/pages/lectures/express-yourself-with-nodejs/index.mdx

```diff
@@ -187,12 +187,12 @@ import audio from "../../../assets/audio/in_the_end.mp3";
     Рассмотрим на примере как же работает Web:
     <ol>
       <li><em>пользователь открыл браузер</em> / <strong>запустил клиент</strong></li>
-      <li><em>пользователь открыл <a href="https://alexandrtovmach.com/images/preloader.svg">ссылку</a></em> / <strong>по HTTP отправил GET запрос на сервер</strong></li>
+      <li><em>пользователь открыл <a href="https://alexandrtovmach.com">ссылку</a></em> / <strong>по HTTP отправил GET запрос на сервер</strong></li>
```

Comment:
Change link to this
https://binary-studio-academy.github.io/stage-2/static/ryu-845240705695b54d820b710eac8d0d5d.png